### ISSUE TITLE
Updating the game template / runners?

### ISSUE THREAD

**Issue by fabianhemmert** [flutter/samples#1542]
Hello! Thank you for doing this – great samples!

I'm using a ~4 months old version of the game template for a project. It recently started throwing deprecation warnings:

```
/Applications/flutter/.pub-cache/hosted/pub.dartlang.org/games_services-2.0.8/macos/Classes/SwiftGamesServicesPlugin.swift:67:25: warning: 'GKScore' was deprecated in macOS 11.0: Replaced by GKLeaderboardScore
    let reportedScore = GKScore(leaderboardIdentifier: leaderboardID)
                        ^
/Applications/flutter/.pub-cache/hosted/pub.dartlang.org/games_services-2.0.8/macos/Classes/SwiftGamesServicesPlugin.swift:69:5: warning: 'GKScore' was deprecated in macOS 11.0: Replaced by GKLeaderboardScore
    GKScore.report([reportedScore]) { (error) in
    ^

/Applications/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_core-1.20.1/macos/Classes/FLTFirebaseCorePlugin.m:229:50: warning: incompatible pointer to integer conversion sending 'NSNumber * _Nonnull __strong' to parameter of type 'BOOL' (aka 'signed char') [-Wint-conversion]
    [firebaseApp setDataCollectionDefaultEnabled:enabled];
                                                 ^~~~~~~
In module 'FirebaseCore' imported from /Applications/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_core-1.20.1/macos/Classes/FLTFirebasePlugin.h:10:
    BOOL dataCollectionDefaultEnabled;
         ^
1 warning generated.
```

`

I'd like to take care of this, so I tried to simply update the
```

    macos
    ios
    android
```

folders of my project with the updated versions from this repository, but that caused more harm, so I went back to my old version. 

It might be related – the next warning thrown is this:

`warning: The macOS deployment target 'MACOSX_DEPLOYMENT_TARGET' is set to 10.10, but the range of supported deployment target versions is 10.13 to 13.1.99. (in target 'PromisesObjC' from project 'Pods')`

What can I do? Thank you!
Fabian

**Comment by domesticmouse** [comment]
Hey Filip can you take a look at this? Thanks!

**Comment by filiph** [comment]
Hi Fabian,

I'd say you're up to something with the deployment target. It's hard for me to figure out what's wrong remotely, but maybe all that's needed is to do a change similar to [this one](https://github.com/flutter/samples/pull/1425/files#diff-6b03ded03e743a098ca3f48c594bcba8a2529a02db21bfbb6d7a44a691da2d83).

If that doesn't work, a "nuclear" solution is to delete the `macos` folder and the run `flutter create .` in your project's directory. (Assuming you have version control so you can go back...)

**Comment by fabianhemmert** [comment]
Hi Filip,

Thank you very much, that solved it – it was the deployment target.

One thing I had to do on the way was this, as my new deployment target wasn't propagated to everywhere, apparently:

https://levelup.gitconnected.com/how-to-fix-your-flutter-macos-target-mismatch-bc55424b7c77

Merry Christmas, everyone, and thank you again!
Fabian
